src/NServiceBus.AcceptanceTests/Performance/TimeToBeReceived/When_TimeToBeReceived_has_expired.cs

```diff
@@ -39,10 +39,30 @@ public Task Handle(MyMessage message, IMessageHandlerContext context)
                     return Task.FromResult(0);
                 }
             }
+
+            class DelayReceiverFromStarting: IWantToRunWhenBusStartsAndStops
+            {
+                /// <summary>
+                /// Method called at startup.
+                /// </summary>
+                public async Task Start(IBusSession session)
+                {
+                    await session.SendLocal(new MyMessage());
+                    await Task.Delay(TimeSpan.FromSeconds(5));
```

Comment:
That does not control the length of the test, the test witll run for https://github.com/Particular/NServiceBus/pull/3297/files#diff-4468ef2934ac7be264be732a2c5fd6ebR16, and I have not changed that.